packages/product-json-to-csv/package.json

```diff
@@ -23,28 +23,36 @@
     "email": "williams.omayuku@commercetools.de"
   },
   "main": "lib/main.js",
+  "bin": {
+    "product-json-to-csv": "bin/product-json-to-csv.js"
+  },
   "files": [
+    "bin",
     "lib"
   ],
   "scripts": {
     "build": "babel src --out-dir lib"
   },
   "dependencies": {
     "@commercetools/api-request-builder": "^3.2.0",
-    "@commercetools/get-credentials": "^2.1.0",
+    "@commercetools/get-credentials": "^2.1.2",
     "@commercetools/sdk-client": "^1.5.0",
     "@commercetools/sdk-middleware-auth": "^3.4.0",
     "@commercetools/sdk-middleware-http": "^2.2.0",
     "@commercetools/sdk-middleware-user-agent": "^1.2.0",
+    "JSONStream": "^1.3.2",
     "archiver": "^2.1.0",
     "bluebird": "^3.5.0",
     "flat": "^4.0.0",
     "highland": "^2.11.1",
     "json2csv": "^3.11.5",
     "lodash": "^4.17.4",
+    "pino": "^4.10.2",
```

Comment:
Hell, yeah <3! I love pino. Next up comes https://github.com/lrlna/pino-colada

Reply:
I checked it out! ❤️